tests/helpers/test_unit_system.py

```diff
@@ -0,0 +1,125 @@
+"""Test the unit system helper."""
+import unittest
+
+from homeassistant.helpers.unit_system import (
+    UnitSystem,
+    METRIC_SYSTEM,
+    IMPERIAL_SYSTEM,
+    TYPE_LENGTH,
+    TYPE_TEMPERATURE,
+    TYPE_MASS,
+    TYPE_VOLUME,
+)
+from homeassistant.const import (
+    LENGTH_METERS,
+    LENGTH_KILOMETERS,
+    MASS_GRAMS,
+    VOLUME_LITERS,
+    TEMP_CELSIUS,
+)
+
+SYSTEM_NAME = 'TEST'
+INVALID_UNIT = 'INVALID'
+
+
+class TestUnitSystem(unittest.TestCase):
+    """Test the unit system helper."""
+
+    def test_invalid_units(self):
+        """."""
```

Comment:
Sorry - following some of the other unit tests - I'm more than happy to add documentation to these.